src/metabase/metabot.clj

```diff
@@ -104,26 +106,26 @@
   [& _]
   (let [cards (with-metabot-permissions
                 (filterv mi/can-read? (db/select [Card :id :name :dataset_query], {:order-by [[:id :desc]], :limit 20})))]
-    (str "Here's your " (count cards) " most recent cards:\n" (format-cards cards))))
+    (tru "Here's your " (count cards) " most recent cards:\n" (format-cards cards))))
```

Comment:
You'll need to switch this to the parameterized '{0}` syntax, i.e. 
``` clojure
(tru "Here's your {0} most recent cards:\n{1}" (count cards) (format-cards cards))
```

Reply:
Oooohhh, gotcha. I stupidly thought those `\n`s were newlines. 😆